### PR TITLE
Init models

### PR DESCRIPTION
**Pull request by dhhIntern**
https://edu-redmine.sun-asterisk.vn/issues/25512

### CODE REVIEW

**Comment by dhhIntern** [comment]
image

**Comment by dhhIntern** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
1 pull 1 commit thôi nhé

**Comment by buithikieu11** [comment]
Ready

**Comment by buithikieu11** [comment]
ready

File: test/models/course_subject_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (vanvtt-0952) [inline test/models/course_subject_test.rb]: Xóa mấy file test/models đi bé nhé

Comment (buithikieu11) [reply test/models/course_subject_test.rb]: Dạ em xóa rồi ạ

**Comment by dhhIntern** [comment]
ready

**Pull request merged**